pkg/kubelet/dockershim/libdocker/kube_docker_client.go

```diff
@@ -183,7 +174,7 @@ func (d *kubeDockerClient) RemoveContainer(id string, opts dockertypes.Container
 func (d *kubeDockerClient) inspectImageRaw(ref string) (*dockertypes.ImageInspect, error) {
 	ctx, cancel := d.getTimeoutContext()
 	defer cancel()
-	resp, _, err := d.client.ImageInspectWithRaw(ctx, ref, true)
+	resp, _, err := d.client.ImageInspectWithRaw(ctx, ref)
```

Comment:
The third parameter is never used (https://github.com/moby/moby/pull/25368), so it's safe to just drop it in the new API.